exonum-java-binding/testkit/src/test/java/com/exonum/binding/testkit/TestKitTest.java

```diff
@@ -180,8 +180,7 @@ void createTestKitWithInvalidArtifactThrows(@TempDir Path directory) throws Exce
         .withService(ARTIFACT_ID, SERVICE_NAME, SERVICE_ID, SERVICE_CONFIGURATION)
         .withArtifactsDirectory(artifactsDirectory);
     RuntimeException thrownException = assertThrows(exceptionType, testKitBuilder::build);
-    assertThat(thrownException.getMessage())
-        .contains("Failed to load the service from ", invalidArtifactFilename);
+    assertThat(thrownException.getMessage()).contains("Unable to create blockchain instance");
```

Comment:
Are there any more specific details in the message? It doesn't tell what is wrong.